realm/realm-jni/src/io_realm_internal_Group.cpp

```diff
@@ -289,3 +289,23 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_Group_nativeEquals(
     } CATCH_STD()
     return false;
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_Group_nativeIsEmpty(
+    JNIEnv*, jobject, jlong nativeGroupPtr)
+{
+    Group* grp = G(nativeGroupPtr);
+    const std::string table_prefix = "class_";
```

Comment:
Currently Table.java also has a constant with this: `Table.TABLE_PREFIX`. Having it as an accessible constant from C++ would probably be nice, so we only have it defined 1 place.